scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/JobData.java

```diff
@@ -27,139 +34,56 @@
 import org.ow2.proactive.scheduler.job.InternalTaskFlowJob;
 import org.ow2.proactive.scheduler.job.JobIdImpl;
 import org.ow2.proactive.scheduler.job.JobInfoImpl;
-import org.hibernate.annotations.NamedQueries;
-import org.hibernate.annotations.NamedQuery;
-import org.hibernate.annotations.OnDelete;
-import org.hibernate.annotations.OnDeleteAction;
-import org.hibernate.annotations.Parameter;
-import org.hibernate.annotations.Type;
-import org.hibernate.type.SerializableToBlobType;
 
 
 @Entity
-@NamedQueries( {
```

Comment:
%!@#* Eclipse Formatter!!! we really have to use a coding template that works same for IntelliJ and Eclipse.